src/stories/SelectPanel.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import {Meta} from '@storybook/react'
+import React, {useCallback, useState} from 'react'
+import {theme, ThemeProvider} from '..'
```

Comment:
If you run `yarn run dist` does it warn about a circular reference? If so, I’d suggest `import`’ing from a location other than `..`.

Reply:
Nope, doesn't seem to care since it's in the tests